File: Java/Car.java
class Car{
    private Integer id;
    private String license;
    private Account driver;
    private Integer passenger;
    
    public Car(String license, Account driver, Integer passenger ){
        this.license = license;
        this.driver = driver;
        if (passenger != 4){
        System.out.println("los pasajeros tienen que ser 4");
        }
        this.passenger = passenger;
        System.out.println("Constructor del carro"); 
    }

    public void setPassenger(Integer passenger){
     
    }
    public Integer getPassenger(){
        return this.passenger;
    }


    public void setLicense(String license){
        this.license=license;
    }
    public String getLicense(){
        return this.license;
    }

    void printDataCar(){
        if (passenger != null){
            System.out.println("licencia " + license + " conductor " + driver +  " pasajero " + passenger );
            }else{
                System.out.println("error de cupo");
            }
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

}

File: Python/car.py
from account import Account

class Car (Account):
    __id= int
    __license = str
    __driver = Account("","","","")
    __passenger = int

    def __init__(self, id, driver ,license,passenger):
        if passenger == 4 or passenger == 6:
            self.__id =id
            self.__license=license
            self.__driver=driver
            self.__passenger = passenger
        else:
            print(" el numero de pasajeros es invalido")
        
    def printDataCar(self):
        if self.__passenger == 4 or self.__passenger == 6:
            print("id carro: " + str(self.__id) + 
            "licencia carro: " + self.__license +
            "numero pasajeros: " + str(self.__passenger))
            return self.__driver.printAccountData()
        else:
            print(" el numero de pasajeros es invalido")

